app/models/treatment.rb

```diff
@@ -0,0 +1,8 @@
+class Treatment < ApplicationRecord
+  #belongs_to :exams
+  
+  validates :title, presence: true
+  validates :description, presence: true
+  validates :establishment, presence: true
+  validates :type, presence: true
```

Comment:
```suggestion
  validates :title, :description, :establishment, :type, presence: true
```

Da pra fazer tudo isso na mesma linha com o validates